doc/architecture/trace_file_mcap_format.adoc

```diff
@@ -0,0 +1,121 @@
+ifndef::include-only-once[]
+:root-path: ../
+include::{root-path}_config.adoc[]
+endif::[]
+= MCAP Format
+
+== General Requirements
+- Must comply with the https://mcap.dev/spec[MCAP format specification] version `0x30`
+- Must allow other non-OSI data to be present in the MCAP file
+- Message records must be written into `chunk records` for indexed files
+- Only OSI top-level messages containing a timestamp field are permitted to be directly stored in MCAP channels
+- Must contain only a single scenario with a unique global time
+- An MCAP file is considered a single dataset
+
+== Schema
+- `name` field: Full message type name, including package (e.g., `osi3.SensorData`)
```

Comment:
I understand the [protobuf documentation](https://protobuf.dev/reference/cpp/api-docs/google.protobuf.descriptor/#Descriptor.full_name.details) as if this is defined.  But to be more precise I will change it to  `fully-qualified name of the protobuf message type`

> Especially, because you used "OSI3::SensorData" in line 60 (not in the same context but anyways).

There I meant the [osi3::SensorData Struct](https://opensimulationinterface.github.io/osi-antora-generator/asamosi/latest/gen/structosi3_1_1SensorData.html). But it makes sense to simply write  "A channel containing OSI `SensorData` messages" and circumvent this. I will change line 60